docker/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# syntax = docker/dockerfile:experimental
+#
+# This file can build images for cpu and gpu env. By default it builds image for CPU.
+# Use following option to build image for cuda/GPU: --build-arg BASE_IMAGE=nvidia/cuda:10.1-cudnn7-runtime-ubuntu18.04
+# Here is complete command for GPU/cuda - 
+# $ DOCKER_BUILDKIT=1 docker build --file Dockerfile.cpu --build-arg BASE_IMAGE=nvidia/cuda:10.1-cudnn7-runtime-ubuntu18.04 -t torchserve:latest .
```

Comment:
Might need to update this from Dockerfile.cpu to Dockerfile

Reply:
Done.

Reply:
@seemethere Can you confirm if this meets your expectations for requested changes and then resolve this conversation?